contrib/registry/consul/client.go

```diff
@@ -71,13 +71,15 @@ func (d *Client) Register(ctx context.Context, svc *registry.ServiceInstance, en
 	addresses := make(map[string]api.ServiceAddress)
 	var addr string
 	var port uint64
```

Comment:
`Addr, port` can actually be removed, and the default asr.Checks = {}.

Reply:
Do not use health check?